GoodDollar/GoodDAPP
# (Feature) Service worker app update

# Description

Show popup and reload app on service worker update

- removed skipwaiting
- removed caching strategy so default is cachefirst
- add caching of external google fonts
- add skipWaiting on press OK button in dialog
- add location.reload when SW activated after skipping

About #795 and #999 


# How Has This Been Tested?

1.  add `REACT_APP_SERVICE_WORKER=true` to env
2. restart app
3. open several windows of applications
4. open check service worker in developers tools in chrome
5. change file public/sw-dev.js (add for ex. `const version=1`)
6 reload app. and check service worker status. it should be a new version of service worker which wait for `skip waiting`. in dapp you should see dialog of new version.
7 click OK. it should setup a new version of service worker and reload all tabs with dapp in chrome


# Checklist:
